gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Thu Apr 30 00:06:06 MSK 2020
```

Comment:
I see no reason to change this file here.

Reply:
It wasn't me. It was IDEA :/